examples/aws-sso/Earthfile

```diff
@@ -0,0 +1,87 @@
+VERSION 0.7
+FROM amazon/aws-cli
+ARG --global EARTHLY_CI=false
+
+target:
+  BUILD +run --sso_region="us-east-2" --region="us-east-2" --args="sts get-caller-identity"
+
+python:
+  FROM python:3
+  WORKDIR /workspace
+
+ubuntu:
+  FROM ubuntu:jammy
+  WORKDIR /workspace
+  RUN apt update -y
+  RUN apt install -y curl unzip
+
+login:
+  IF [ "$EARTHLY_CI" = "false" ]
+    ARG --required sso_region
+    FROM +python
+    # https://github.com/benkehoe/aws-export-credentials
+    RUN pip install aws-export-credentials
+    BUILD +dev.login.open --sso_region=$sso_region
+    COPY +dev.login/aws /root/.aws
+    RUN aws-export-credentials --profile default --credentials-file-profile default
+    SAVE ARTIFACT /root/.aws/credentials /credentials
+  ELSE IF [ "$EARTHLY_CI" = "true" ]
+    RUN mkdir -p /root/.aws/
+    RUN --secret AWS_ACCESS_KEY_ID --secret AWS_SECRET_ACCESS_KEY --secret AWS_SESSION_TOKEN printf "[default]\naws_access_key_id=$AWS_ACCESS_KEY_ID\naws_secret_access_key=$AWS_SECRET_ACCESS_KEY\naws_session_token=$AWS_SESSION_TOKEN\n" >> /root/.aws/credentials
+  ELSE
+    RUN echo "bad value for EARTHLY_CI" && exit 1
+  END
+  SAVE ARTIFACT /root/.aws/credentials /credentials
+
+dev.login.open:
+  ARG --required sso_region
+  LOCALLY
```

Comment:
since this is `LOCALLY`, can you include a test for deps? e.g.

```
RUN command -v python3 >/dev/null || (echo "python3 is needed locally" && exit 1)
```

just in case the user doesn't have python.